soft4pes/model/machine/base_machine.py

```diff
@@ -13,11 +13,11 @@ class BaseMachine:
 
     Parameters
     ----------
-    Vr : float
+    V_R_SI : float
```

Comment:
notation needs to be updated as per my previous comment (add ``m`` to the subscript)

Reply:
Added in next commit